config.go

```diff
@@ -8,6 +8,7 @@ import (
 type Config struct {
 	LogLevel             string `yaml:"log_level"`
 	Listen               string `yaml:"listen"`
+	EnableServerTLS      bool   `yaml:"enable_server_tls"`
```

Comment:
Please also add this to the config-template.yml with a default value of `true`. Otherwise the default behavior changes